src/plugins/intel_cpu/src/nodes/deconv.cpp

```diff
@@ -425,8 +425,8 @@ void Deconvolution::initPaddingR(const Shape &inShape, const Shape &outShape) {
         int dst = inShape.getStaticDims()[2 + i];
 
         krn = (krn - 1)*(dilation[i] + 1) + 1;
-        int calc_dst = (src - krn + paddingL[i]) / stride[i] + 1;
-        paddingR[i] = (dst - calc_dst) * stride[i];
+        float calc_dst = static_cast<float>(src - krn + paddingL[i]) / stride[i] + 1.;
+        paddingR[i] = round((dst - calc_dst) * stride[i]);
     }
```

Comment:
since calc_dst is just a temp variable and it may not be integer, changing from `int` to `float` only makes the rounding error smaller but it would be still there ( for example even `float` format would introduce error when representing 1/3), so why not substitute `(src - krn + paddingL[i]) / stride[i] + 1` into the next equation and eliminate the division totally like this:

```
        paddingR[i] = ((src - krn + paddingL[i]) / stride[i] + 1 - calc_dst) * stride[i];
                    = (src - krn + paddingL[i]) + (1 - calc_dst) * stride[i];
```

What do you think?

Reply:
yes,  use * instead of / is a good idea, I will try it